ui/license_page.go

```diff
@@ -0,0 +1,129 @@
+package ui
+
+import (
+	"gioui.org/layout"
+	"github.com/planetdecred/godcr/ui/decredmaterial"
+	"github.com/planetdecred/godcr/ui/values"
+)
+
+const PageLicense = "License"
+
+type licensePage struct {
+	theme         *decredmaterial.Theme
+	common        *pageCommon
```

Comment:
```suggestion
	*pageCommon
```
Make `pageCommon` part of this struct and you'd be able to access the properties directly.